modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspacePlugin.java

```diff
@@ -112,14 +112,6 @@ public void doCall(Project project) {
 
 			});
 
-		String absolutePath = rootDir.getAbsolutePath();
```

Comment:
Did you test this change on Windows machine?

Reply:
I have not. Is it necessary to do OS-specific checks when you just work the `Path` objects instead of strings?

Reply:
Ok I sent a separate pull to Peter with just this change so we can check on Windows.